lib/core/topologies/read_preference.js

```diff
@@ -12,6 +12,10 @@
  * @return {ReadPreference}
  */
 const ReadPreference = function(mode, tags, options) {
```

Comment:
While we're here, should we convert `ReadPreference` to a class?

Reply:
I'd prefer to leave this work for another ticket and focus on Command Refactor work for now.